catalog-info.yaml

```diff
@@ -20,6 +22,12 @@ metadata:
     - url: https://app.ddog-gov.com/sb/f327ad72-c02a-11ec-a50a-da7ad0900007-a1dfad12b43924de9033ad82897aadab
       title: Datadog - VSP - External Service Performance Indicators
       icon: dashboard
+    - url: https://grafana.vfs.va.gov/
+      title: Grafana
+      icon: dashboard
+    - url: http://sentry.vfs.va.gov/organizations/vsp/
```

Comment:
If this board isn't the correct one to link to let me know I can revise -- or add multiple.